core/src/test/java/com/datastax/oss/driver/internal/core/metadata/DefaultTopologyMonitorTest.java

```diff
@@ -407,6 +407,9 @@ public void should_skip_invalid_peers_row_v2(String columnToCheck) {
       case "host_id":
         when(peer2.getUuid(columnToCheck)).thenReturn(null);
         break;
+      case "data_center":
+        when(peer2.getUuid(columnToCheck)).thenReturn(null);
```

Comment:
This should be `when(peer2.getDatacenter(columnToCheck)).thenReturn(null);`

Reply:
nice catch! thanks!